babel-preset/plugins/__tests__/rewrite-modules-test.js

```diff
@@ -33,6 +33,20 @@ describe('rewrite-modules', function() {
         expect(result.code).toEqual('require(\'test/test\');');
       });
 
+      it('should replace the prefix on type imports', function() {
+        let result = babel.transform(
+          'import type Test from \'test\';',
```

Comment:
Use backticks for single line string will cause warnings in eslint quotes rule: `Strings must use singlequote.`

I changed it to `'import type Test from "test"'` to avoid escaping. I saw it is used this way in `inline-requires-test.js` too.